src/actions/assetsActions.js

```diff
@@ -46,6 +46,7 @@ import { ADD_COLLECTIBLE_TRANSACTION, COLLECTIBLE_TRANSACTION } from 'constants/
 import { PAYMENT_NETWORK_SUBSCRIBE_TO_TX_STATUS } from 'constants/paymentNetworkConstants';
 import { ERROR_TYPE } from 'constants/transactionsConstants';
 import { SET_TOTAL_ACCOUNT_CHAIN_CATEGORY_BALANCE, SET_FETCHING_TOTALS } from 'constants/totalsConstants';
+import { NetworkNames } from 'etherspot';
```

Comment:
We keep library imports at the top

Reply:
This was automatically imported - hence i missed it! It's now redundant and removed 👍